I have modified the template to allow modifying the 'Curriculum Vitae' text

kleinjoshuaa opened the pull request:
This would allow users to use this text for something else  (i.e. 'Resume') or omit it entirely. 

See the latex markup:
![test_text](https://cloud.githubusercontent.com/assets/1207274/7105621/3d464a2a-e0f0-11e4-9dcd-cd68a2080d94.png)

and the output:
![output](https://cloud.githubusercontent.com/assets/1207274/7105622/40e3de0e-e0f0-11e4-83c8-333b0626ed6e.png)

kleinjoshuaa committed:
I have modified the template to allow modifying the 'Curriculum Vitae'
text underneath the name for those not using this template for a CV
 Changes to be committed:
	modified:   fausto.en.tex
	modified:   nicola.en.tex
	modified:   nicola.it.tex
	modified:   tccv.cls

ntd commented:
Can you refactor your pull request to accept the text as optional argument, defaulting to `Curriculum vitae`? This will not allow its omission but it is backward compatible and IMO just straighter. At the end your example should be:

```
\part[Test Text]{Joshua Klein}
```

If omitting the text is desirable I'd prefer to split `\part` into two new commands (I should have used a new command instead of `\part` in the first place).

kleinjoshuaa committed:
Modified to make command optional, defaulting to keep backwards compatibiliy
 Changes to be committed:
	modified:   fausto.en.tex
	modified:   nicola.en.tex
	modified:   nicola.it.tex
	modified:   tccv.cls

kleinjoshuaa committed:
Modified Comments
 Changes to be committed:
	modified:   tccv.cls

kleinjoshuaa commented:
Hello, updated to make the command optional 

Let me know your thoughts, or if this is ready to be committed.

 This is with the argument
![image](https://cloud.githubusercontent.com/assets/1207274/7107200/fa63bb34-e12b-11e4-8bf8-ff06acfd340b.png)
Renders to:
![image](https://cloud.githubusercontent.com/assets/1207274/7107204/0201aafe-e12c-11e4-96d9-4a7b737211ca.png)

And without the optional argument
![image](https://cloud.githubusercontent.com/assets/1207274/7107210/1a38668a-e12c-11e4-96eb-8331aa2f6ee5.png)

Renders to:
![image](https://cloud.githubusercontent.com/assets/1207274/7107213/23798300-e12c-11e4-8b2b-5ea157ccb8c8.png)

ntd commented:
Much better. Thanks for the pull request: merged.
